spec/features/upload_lead_image_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+RSpec.feature "Upload a lead image" do
+  scenario "User uploads a lead image" do
+    given_there_is_a_document
+    when_i_visit_the_summary_page
+    then_i_see_there_is_no_lead_image
+    when_i_visit_the_lead_images_page
+    then_i_should_see_no_images_available
+    and_i_upload_a_new_image
+    and_i_fill_in_the_metadata
+    then_i_should_be_able_to_see_the_image
```

Comment:
We're implicitly checking it's the lead image - perhaps the step should express that too :-).